src/lib/stores/marketplace.ts

```diff
@@ -65,7 +65,7 @@ export const marketplace: MarketplaceTemplate[] = [
         vcsProvider: 'github',
         providerRepositoryId: 'templates',
         providerOwner: 'appwrite',
-        providerBranch: '0.1.*',
+        providerBranch: '0.1.x',
```

Comment:
In 1.5, this must remain 0.1.*. There is a special check for it to support tag releases.
From 1.6, we will only use tag releases.

Reply:
In 1.6 this will become 0.2.*. I am still working towards 0.2 release before adding it to Appwrite 1.6